### DISCUSSION TITLE
Search fucionality in chat?

### DISCUSSION THREAD

**Discussion by snspinn** [zed-industries/zed#20903] [Feature Requests]
Is there a plan to add search functionality in the chat?

Might make chat more accessible if I can check if anyone has asked a question similar to mine, recently.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
